blocks/sensors.js

```diff
@@ -48,3 +48,28 @@ Blockly.Blocks['sensors_get_covered'] = {
     this.setHelpUrl('http://www.example.com/');
   }
 };
+
+Blockly.Blocks['light_sensor_value'] = {
+  init: function() {
+    this.appendDummyInput()
+        .appendField(new Blockly.FieldDropdown([["left","LEFT"], ["right","RIGHT"]]), "SENSORS")
+        .appendField("light sensor value");
+    this.setOutput(true, "Number");
+    this.setTooltip("Get the brightness value from a light sensor. 0 is darkest; 1023 is brightest.");
+    this.setColour(Blockly.Blocks.sensors.HUE);
+    this.setHelpUrl("https://docs.rovercode.com/blocks/light-sensor-value");
+  }
+};
+
+Blockly.Blocks['button_press'] = {
+  init: function() {
+    this.appendDummyInput()
+        .appendField("button")
+        .appendField(new Blockly.FieldDropdown([["A","A"], ["B","B"]]), "BUTTON")
+        .appendField("has been pressed");
```

Comment:
We have to make a decision how to handle the button state. The micro:bit only provides the `pressed` action, so we can't keep track of pressed/not-pressed in the UI.
I propose that we set the state to `hasBeenPressed` when a button is pressed, and we keep it like that until this block queries its state, when we'll clear it.